tests/backends/test_Windows.py

```diff
@@ -58,8 +29,22 @@ def test_read_utf8_password(self):
         username = "keyring"
         password = "utf8-test" + UNICODE_CHARS
 
-        self.set_utf8_password(service, username, password)
```

Comment:
I broke that when I removed `pywin32-ctypes`.  Here's the thing though, I think purely by chance, the test used to work due to the choice of an odd (rather than even) length `password`.  If you change

```
password = "utf8-test" + UNICODE_CHARS
```

to

```
password = "utf8-test0" + UNICODE_CHARS
```

at https://github.com/jaraco/keyring/blob/04269447ae7d30d3e31b9122fcd0e0358c87eb51/tests/backends/test_Windows.py#L59, you'll see the default `'utf-16'` decode "work", but not with the desired results.

The only way I could think to make the desired results work was to add an `attribute` to the Credentials Manager entry, explictly stating the encoding uses for the password.  

I should have done a better job documenting my concern, and not leaving this test saying one thing and doing another.